packages/terra-graphs-docs/src/example-datasets/popup.js

```diff
@@ -206,9 +209,10 @@ export const loadTrackPopup = (onCloseCB, key, value) => {
     removeOldPopup();
     const path = renderPopup(onCloseCB);
     const pair = path.append("g");
-    if (value.tasks && value.tasks.length > 0) {
+    const { tasks, activities, events, actions } = value;
+    if (tasks && tasks.length > 0) {
```

Comment:
an alternative is just checking the length property, because 0 is a falsey value
```suggestion
    if (tasks && tasks.length) {
```

Reply:
Yes you are right, but I think it would be more readable when it is tasks.length>0. Correct me if I am wrong.